graph/src/components/store.rs

```diff
@@ -1179,7 +1177,7 @@ impl Store for MockStore {
         self: Arc<Self>,
         _subgraph_id: &'a SubgraphDeploymentId,
         _indexer: &'a Option<Address>,
-        _block_hash: H256,
+        _block: &'a EthereumBlockPointer,
```

Comment:
`EthereumBlockPointer` is `Copy` so there is no need to use a reference.

Reply:
Cool, updated to avoid the reference and things still compile and work. Thanks!